spacecmd/src/spacecmd/configchannel.py

```diff
@@ -866,12 +866,20 @@ def do_configchannel_addfile(self, args, update_path=''):
             if options.directory:
                 if 'contents' in file_info:
                     del file_info['contents']
+                if 'contents_enc64' in file_info:
+                    del file_info['contents_enc64']
+                if 'binary' in file_info:
+                    del file_info['binary']
 
-            self.client.configchannel.createOrUpdatePath(self.session,
-                                                         options.channel,
-                                                         options.path,
-                                                         options.directory,
-                                                         file_info)
+            try:
+                self.client.configchannel.createOrUpdatePath(self.session,
+                                                             options.channel,
+                                                             options.path,
+                                                             options.directory,
+                                                             file_info)
+            except xmlrpclib.Fault as exc:
+                logging.error(exc)
```

Comment:
How does the output look like from the user perspective in case such a Fault is raised, after this patch?

Reply:
this is an example of entire output
`spacecmd {SSM:0}> configchannel_updatefile -c salt_from_cli -p /root/conf -f /root/conf -d
Path:            /root/conf
Directory:       True
Owner:           root
Group:           root
Mode:            0755
Binary:          False
SELinux Context: 

Is this ok [y/N]: y
ERROR: <Fault 1233: 'redstone.xmlrpc.XmlRpcFault: **Directories/Symlinks are not supported in state channel.**'>
`

Reply:
Sounds good!